### ISSUE TITLE
Constraints

### ISSUE THREAD

**Issue by jetgeo** [ISO-TC211/ISO19110#6]
Constraints are written both in the attribute notes and as OCL constraints. In the diagrams, they are presented as notes, linked to the classes. I would suggest changing this, according to best practices. They should rather be  presented as part of the classes, showing just the text, not the OCL. I have done this change, but it can be reverted if you disagree.

**Comment by jetgeo** [comment]
I have tried to harmonize the constraints in the model with the requirements in Table C.1. I have suggested some changes in table C.1 in the document ISO_FDIS_19110_(E)dd_smr:
• Row 2 & 3: Changed from “is Mandatory” to “is mandatory”
• Row 4: Removed parenthesis (ref next)
• Row 5: Changed affected class from FC_CarrierOfCharacteristics to FC_PropertyType. The expression is referring to the role featureCatalogue. This role does not exist on FC_CarrierOfCharacteristics, only on the subclass FC_PropertyType.
• Row 10 & 11: Changed affected class from FC_BoundFeaturAssociation to FC_BoundAssociationRole (this is the name in the model)
• Row 12 &13: The expressions are duplicates, and none of the text versions are clear. I’m not sure that I understand this, but I believe the idea is that the role featureType shall be globaly defined, not through FC_PropertyType. I suggest to keep only one of the requirements, and change the text to “featureType shall be global”.
